src/wizard/obtain_source.jl

```diff
@@ -352,6 +352,30 @@ function get_name_and_version(state::WizardState)
     end
 end
 
+@enum Compilers C=1 Go Rust
+function get_compilers(state::WizardState)
+    while state.compilers === nothing
+        compiler_descriptions = Dict(C => "C/C++/Fortran", Go => "Go", Rust => "Rust")
+        compiler_symbols = Dict(Int(C) => :c, Int(Go) => :go, Int(Rust) => :rust)
+        terminal = TTYTerminal("xterm", state.ins, state.outs, state.outs)
+        result = nothing
+        while true
+            select_menu = MultiSelectMenu([compiler_descriptions[i] for i in instances(Compilers)])
+            select_menu.selected = Set([Int(C)])
+            result = request(terminal,
+                             "Select compilers for the project",
+                             select_menu
+                             )
+            if isempty(result)
+                println("Must select at least one platform")
+            else
+                break
+            end
+        end
+        state.compilers = map(c -> compiler_symbols[c], collect(result))
+    end
+end
```

Comment:
Is there some way we can add `preferred_gcc_version`/`preferred_llvm_version` here?

Reply:
Yes, I was thinking about adding all these settings here. However I'll be a bit busy these days 🙂